notebooks/ch15/bias_cv.md

```diff
@@ -0,0 +1,34 @@
+
+## Cross-Validation
+
+In the previous section, we observe that the we need a way to simulate test error to more accurately manage the bias-variance trade off. Cross-validation provides a method of estimating our model error using a single observed dataset by separating data used for training from the data used for model selection and final accuracy. 
+
+## K-Fold Cross-Validation
+
+In Chapter 14.3, we mentioned the method of the **train-validation split**, which can be used to simulate test error through the validation set. However, this method is prone to high variance of the validation error because the evaluation of the error may depend heavily on which points end up in the training and validation sets.
+
+To tackle this problem, we can run the train-validation split multiple times on the same dataset. The dataset is divided into *k* equally-sized subsets subsets (*$k$ folds*), and the train-validation split is repeated k times. Each time, one of the *k* folds is used as the validation set, and the reaming *k-1* folds are used as the training set. We report the model's final validation error as the average of the $ k $ validation errors from each trial. This method is called **k-fold cross-validation**. The biggest advantage of this method is that every data point is used for validation exactly once, and for training *k-1* times.
```

Comment:
Nit: Use $ k $ instead of k in the second sentence.
Typo: "subsets" is included twice (second sentence)
Typo: "reaming" instead of "remaining"
Nit: No comma needed before and in the last sentence.